src/s390_ecc.c

```diff
@@ -1750,9 +1750,14 @@ struct {				\
 					rc = s390_kdsa(fc, param.buff, NULL, 0);
 				} while (rc);
 			} else {
+				/* Also create a random k-value for the non-deterministic
+				 * case as this number will be XORed with an internally
```

Comment:
Not sure if it is XORed.... The POP states: "_specified random number is hashed with a CPU-generated random number_", so I would not mention XOR here.

The same comment would belong to line 1760.... but having it twice is certainly not really needed.

Reply:
ok, changed comment. And we had it even 3 times. I added a "same for p384 and p521" and removed 2 comments.

Reply:
also change the commit description please, you also mention XOR there.